uxarray/core/dataset.py

```diff
@@ -345,3 +348,25 @@ def nearest_neighbor_remap(
             Indicates whether to remap using on spherical or cartesian coordinates
         """
         return _nearest_neighbor_uxds(self, destination_obj, remap_to, coord_type)
+
+    def inverse_distance_weighted_remap(
+        self,
+        destination_obj: Union[Grid, UxDataArray, UxDataset],
+        remap_to: str = "nodes",
+        coord_type: str = "spherical",
```

Comment:
Parameters should match:

https://github.com/UXARRAY/uxarray/blob/086b9e9ede1def5de9c127429846a42f7d5a32fb/uxarray/remap/inverse_distance_weighted.py#L16-L23